topics/assembly/tutorials/chloroplast-assembly/tutorial.md

```diff
@@ -0,0 +1,312 @@
+* Run an alternative polishing tool, such as Racon. This uses the long reads themselves to correct the long-read (Flye) assembly. It would be better to run this tool on the Flye assembly before running Pilon, rather than after Pilon.
+
+# View reads
+
+We will look at the original sequencing reads mapped to the genome assembly. In this tutorial, we will import very cut-down read sets so that they are easier to view.
+
+> ### {% icon hands_on %} Hands-on: Import cut-down read sets
+> 1. Import from [Zenodo](https://zenodo.org/record/3567224) or a data library (ask your instructor):
+>   - FASTQ file with illumina reads: `sweet-potato-chloroplast-illumina-tiny.fastq`
+>   - FASTQ file with nanopore reads: `sweet-potato-chloroplast-nanopore-tiny.fastq`
+>   - Note: these are the "tiny" files, not the "reduced" files we imported earlier.
+>    ```
+>    https://zenodo.org/record/3567224/files/sweet-potato-chloroplast-illumina-tiny.fastq
+>    https://zenodo.org/record/3567224/files/sweet-potato-chloroplast-nanopore-tiny.fastq
+>    ```
+{: .hands_on}
+
+> ### {% icon hands_on %} Hands-on: Map the reads to the assembly
+> * Map the Illumina reads (the new "tiny" dataset) to the `polished-assembly.fasta`, the same way we did before, using bwa mem.
+> * This creates one output file: re-name it `illumina-tiny.bam`
+> * Map the Nanopore reads (the new "tiny" dataset) to the `polished-assembly.fasta`. The settings will be the same, except `Select analysis mode` should be `Nanopore`
+> * This creates one output file: re-name it `nanopore-tiny.bam`
+{: .hands_on}
+
+> ### {% icon hands_on %} Hands-on: Visualise mapped reads
+> 1. **JBrowse genome browser** {% icon tool %}:
+>    - *"Reference genome to display"*: `Use a genome from history`
+>        - *"Select a reference genome"*: `polished-assembly.fasta`
+>    - *"Produce Standalone Instance"*: `Yes`
+>    - *"Genetic Code"*: `11. The Bacterial, Archaeal and Plant Plastid Code`
+>    - *"JBrowse-in-Galaxy Action"*: `New JBrowse instance`
+>    - *"Insert Track Group"*
+>        - *"Insert Annotation Track"*
+>            - *"Track Type"*: `BAM pileups`
+>            - *"BAM track data"*: `nanopore-tiny.bam`
+>            - *"Autogenerate SNP track"*: `No`
+>            - *Leave the other track features as default*
+>        - *"Insert Annotation Track"*.
+>            - *"Track Type"*: `BAM pileups`
+>            - *"BAM track data"*: `illumina-tiny.bam`
+>            - *"Autogenerate SNP track"*: `No`
+>            - *Leave the other track features as default*
+{: .hands_on}
+
+There is one output file: re-name: `assembly-and-reads`
+* Click on the eye icon to view. (For more room, collapse Galaxy side menus with corner < > signs).
+*  Make sure the bam files are ticked in the left hand panel.
+* Choose a contig in the drop down menu. Zoom in and out with + and - buttons.
+
+![read mapping](../../images/jbrowse-read-mapping-zoomed-out.png "Overview of reads mapped back to the assembly")
```

Comment:
if you like, you can embed a real jbrowse here if the data is an ok size? There's an example in https://training.galaxyproject.org/training-material/topics/assembly/tutorials/ecoli_comparison/tutorial.html#visualising-the-genome / https://github.com/galaxyproject/training-material/blob/master/topics/assembly/tutorials/ecoli_comparison/tutorial.md#jbrowse

Reply:
Thanks, sounds good - I will try it now

Reply:
This seems to work now